docs/reference/commandline/dockerd.md

```diff
@@ -828,6 +829,24 @@ To set the DNS search domain for all Docker containers, use:
 $ sudo dockerd --dns-search example.com
 ```
 
+#### Air-gapped registries
```

Comment:
@cpuguy83 To be clear, my suggestion was a name replacement for the registry description and not the artifacts themselves.

But your original point has me a little muddy on this feature's application after giving it some more thought. How do you even push to an air-gapped (isolated) registry? 😆